src/Swarrot/Processor/MemoryLimit/MemoryLimitProcessor.php

```diff
@@ -40,8 +40,9 @@ public function process(Message $message, array $options)
 
         if (null !== $options['memory_limit'] && memory_get_usage() >= $options['memory_limit'] * 1024 * 1024) {
             $this->logger->info(
-                sprintf('[MemoryLimit] Memory limit has been reached (%d MB)', $options['memory_limit']),
+                '[MemoryLimit] Memory limit has been reached (%d MB)',
```

Comment:
`\d MB` should be removed, as this is not a sprintf pattern anymore.

Reply:
See #191